src/lib/bootloader/mbr_update.rb

```diff
@@ -182,13 +180,13 @@ def activatable_partitions(disk)
 
       # do not select swap and do not select BIOS grub partition
       # as it clear its special flags (bnc#894040)
-      devicegraph.disks.with(name: disk.name).partitions.reject do |part|
-        [Storage::ID_SWAP, Storage::ID_BIOS_BOOT].include?(part.id)
+      disk.partitions.reject do |part|
+        [Y2Storage::ID::SWAP, Y2Storage::ID::BIOS_BOOT].include?(part.id)
       end
     end
 
     def extended_partition(disk)
-      part = activatable_partitions(disk).find { |p| p.type == Storage::PartitionType_EXTENDED }
+      part = activatable_partitions(disk).find { |p| p.type == Y2Storage::PartitionType::EXTENDED }
```

Comment:
one disadvantage of symbol is that constant is checked in tests and it failed if there is typo. But not in case of symbol